clang/lib/Sema/SemaDecl.cpp

```diff
@@ -12239,7 +12239,8 @@ bool Sema::CheckFunctionDeclaration(Scope *S, FunctionDecl *NewFD,
 
     if (UsesSM) {
```

Comment:
The comment on line 12232-12233 needs an update.

Reply:
Done.